recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     sha256: dceb15fef556a2775121bcc695561df4ff0e09248e0ce3a2d58f5244a9f61421
 
 build:
-    number: 0
+    number: 1
```

Comment:
Agreed. Unless there are functional changes there is no need for the build number bump.